api/http/api_inventory.go

```diff
@@ -24,12 +24,13 @@ import (
 	u "github.com/mendersoftware/go-lib-micro/rest_utils"
 	"github.com/pkg/errors"
 
+	"regexp"
```

Comment:
Move this to the block with stdlib imports. We try to use the following imports order: stdlib, 3rd party, repo local